airflow/auth/managers/fab/security_manager/override.py

```diff
@@ -103,6 +112,16 @@ class FabAirflowSecurityManagerOverride(AirflowSecurityManagerV2):
     permission_model = Permission
     registeruser_model = RegisterUser
 
+    """ Flask-JWT-Extended """
+    jwt_manager = None
+    """ Flask-OpenID OpenID """
+    oid = None
+    """ OAuth email whitelists """
+    oauth = None
+    """ OAuth email whitelists """
+    oauth_remotes: dict[str, Any]
+    oauth_user_info = None
```

Comment:
I believe the convention is to put docstrings _below_ attributes

https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html

Reply:
My bad, I'll fix it

Reply:
Done